modules/oadp-restic-issues.adoc

```diff
@@ -99,13 +99,43 @@ $ oc get all -n <namespace>
 [id="restic-backup-cannot-be-recreated-after-s3-bucket-emptied_{context}"]
 == Restic Backup CR cannot be recreated after bucket is emptied
```

Comment:
Inconsistent hyphenation in header and text: `recreate` vs `re-create`

I think (but not 100% positive) the IGS discourages hyphens unless they are required for clarity. In this case, 'recreate' is sufficiently clear.